src/controllers/expenseController.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const { findUser: currentUser } = require('../helpers');
+const {
+  getAll,
+  getById,
+  createOne,
+  deleteOne,
+  updateOne,
+  filterByCategory,
+  filterByDate,
+} = require('../services/expensesServices');
+const { findAll } = require('../services/userServices');
+
+const get = (req, res) => {
+  const expenseList = getAll();
+  const { userId, categories, from, to } = req.query;
+
+  let filteredExpenses = filterByCategory(expenseList, categories, userId);
+
+  filteredExpenses = filterByDate(filteredExpenses, from, to);
```

Comment:
Don't filter in the controller, any business logic should be placed in the service
I mean `getAll` should do all work and just return filtered expenses that you can send to the client